### PR TITLE
[RN-781]: ios/android quick links

### PR DESCRIPTION
**Pull request by rastajpa**

### CODE REVIEW

File: src/constants/shortcuts.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import {Platform} from 'react-native';
+import {ShortcutItem} from 'react-native-quick-actions';
+
+export const ShortcutList: Array<ShortcutItem> = [
+  {
+    type: 'buy',
+    title: 'Buy Cryptos',
```
Comment (cmgustavo) [inline src/constants/shortcuts.ts:7]: Just a typo: "Buy Crypto"
Screenshot 2022-12-16 at 13 27 26

**Review by cmgustavo — APPROVED** [review]
Nice work! Tested on iOS and Android
LGTM 👍🏼

**Review by cmgustavo — CHANGES_REQUESTED** [review]
I found an issue specifically on Android. The event `quickActionShortcut` is permanently called every time you enter (or back) to Home. So, if you click on a shortcut function (receive, send, buy, or swap), then you go back to Home, it will return to the view "Select a currency". Also, if you go back, and change the tab, it's fine. But if you return to Home, it will redirect again to "Select a currency".

File: src/constants/shortcuts.ts (hunk 1 above)
Comment (rastajpa) [reply src/constants/shortcuts.ts]: ouch! ty!

**Comment by rastajpa** [comment]
> I found an issue specifically on Android. The event `quickActionShortcut` is permanently called every time you enter (or back) to Home. So, if you click on a shortcut function (receive, send, buy, or swap), then you go back to Home, it will return to the view "Select a currency". Also, if you go back, and change the tab, it's fine. But if you return to Home, it will redirect again to "Select a currency".

thank you, I'm working on a fix

**Review by cmgustavo — DISMISSED** [review]
Worked on Android! 👍🏼

**Review by cmgustavo — APPROVED** [review]
ACK! 🚀

**Review by Gamboster — APPROVED** [review]
LGTM
Nice work!!

**Pull request merged**